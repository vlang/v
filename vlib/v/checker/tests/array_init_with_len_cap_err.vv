_ = []int{len: -1}
_ = []int{cap: -1}
