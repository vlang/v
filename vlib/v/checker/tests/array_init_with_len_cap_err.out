vlib/v/checker/tests/array_init_with_len_cap_err.vv:1:16: error: array len can not be negative
    1 | _ = []int{len: -1}
      |                ~~
    2 | _ = []int{cap: -1}
vlib/v/checker/tests/array_init_with_len_cap_err.vv:2:16: error: array cap can not be negative
    1 | _ = []int{len: -1}
    2 | _ = []int{cap: -1}
      |                ~~
